testapp/home/models.py

```diff
@@ -2,6 +2,10 @@
 from wagtail.core.models import Page
 from wagtail.core.fields import StreamField
 
+from wagtailnhsukfrontend.mixins import (
+    ReviewDate,
```

Comment:
I think this class needs to be called `ReviewDateMixin` instead of just `ReviewDate`.

It's a wagtail convention.